### DISCUSSION TITLE
Replacement word formatting

### DISCUSSION THREAD

**Discussion by andyhaskell** [VickiLanger/OpenGates#12] [Ideas]
For the sections on exclusionary language, do we want to put replacement words in bold or as sub-bullet points? I was thinking that would help keep individual bullet points from getting overly long and harder to read

**Comment by VickiLanger** [comment]
I wanna avoid having it be a remake of http://selfdefined.app. But still thought it was important to include terms not to use. 

To keep each line short, I really would prefer it just be:
~~not this word~~, try [insert a few options] (ref)

Reference should link to a source with explanations and examples if needed.

This isn't going to be exhaustive, http://selfdefined.app does that really well and I think giving 2 or 3 other options is likely enough to make people think. I think that's enough to prompt them to use a different word. I did worry if we didn't give alternatives that people wouldn't want to take the time to look it up. that's why I gave alternatives to begin with.￼

**Comment by VickiLanger** [comment]
Another thought: I'm gonna check out how screen reader friendly it would be to use an ascii arrow instead of "instead, try". I think the strikethrough (gonna see how that sounds on screen reader too) makes it obvious enough that the word is not to be used.

So, it could look like:
~~word~~ → alternative, alternative, alternative 

That arrow is read as: "Rightwards arrow"

**Comment by VickiLanger** [comment]
Interesting, strikethrough isn't generally read at all with a screenreader. When it is, sometimes it will omit the struckthrough text. Of note, other inline text-changing elements also don't get read. [reference](https://www.tempertemper.net/blog/be-careful-with-strikethrough)

For now, we'll go with this:
not ~~word~~ → **alternative**, **alternative**, or **alternative**
